consensus/src/main/java/org/apache/iotdb/consensus/ratis/RatisConsensus.java

```diff
@@ -603,8 +607,8 @@ public Peer getLeader(ConsensusGroupId groupId) {
     if (leaderId == null) {
       return null;
     }
-    TEndPoint leaderEndpoint = Utils.formRaftPeerIdToTEndPoint(leaderId);
-    return new Peer(groupId, leaderEndpoint);
+    int nodeId = Utils.formRaftPeerIdToNodeId(leaderId);
+    return new Peer(groupId, nodeId, null);
```

Comment:
This line used endpoint before, I'm not sure if there exist problem if we replace with nodeId.